cmd/plasmacli/subcmd/eth/challenge.go

```diff
@@ -14,6 +14,7 @@ import (
 	"strconv"
 )
 
+// ChallengeCmd returns the eth challenge command
```

Comment:
For all CMD documentation. I think more detail is needed.
Suggested pattern.
```suggestion
// ChallengeCmd returns the eth challenge command. Use to challenge invalid exits.
```

^Some basic description of what the command does should be added to all cmd documentation imo. Can be a single sentence or two